tools/xcodeproj-shims/ld-stub.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -eu
+
+while :; do
+    test $# -eq 0 && exit 0
+    case $1 in
+        *.dat)
+            echo -n -e '\x00lld\0' > $1
```

Comment:
Maybe a few comments explaining this? At least mention that this is the header of an empty .dat file.

Reply:
done.